torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -458,6 +463,40 @@ std::string stacksToStr(const std::vector<std::string>& stacks, const char* deli
 
 } // namespace
 
+void reportBackendEventToActiveKinetoProfiler(
+    const int64_t start_time_us,
+    const int64_t end_time_us,
+    const int64_t debug_handle,
+    const at::RecordScope scope,
+    const std::string& event_name,
+    const std::string& backend_name) {
+  auto state_ptr = getProfilerTLSState();
+  if (!state_ptr) {
+    return;
+  }
+  auto ctx_ptr = std::make_unique<KinetoObserverContext>();
+  ctx_ptr->correlationId = std::numeric_limits<uint64_t>::max();
+  ctx_ptr->startThreadId = at::RecordFunction::currentThreadId();
+  ctx_ptr->debug_handle = debug_handle;
+
+  /* no support for input shapes now?
```

Comment:
why's this?

Reply:
Mainly to avoid overhead of shape profiling.